packages/dd-trace/src/llmobs/plugins/base.js

```diff
@@ -36,6 +37,8 @@ class LLMObsPlugin extends TracingPlugin {
     // register options may not be set for operations we do not trace with llmobs
     // ie OpenAI fine tuning jobs, file jobs, etc.
     if (registerOptions) {
+      telemetry.incrementLLMObsSpanStartCount({ autoinstrumented: true, kind: registerOptions.kind })
```

Comment:
can we tag the integration name here too like we do in python: https://github.com/DataDog/dd-trace-py/pull/11983/files#diff-e604bc37e976ae7f038f3307b8709878692ec2918d4aeced3e84046af3972720R146

Reply:
oof i totally missed that, yes can do!

Reply:
[added](https://github.com/DataDog/dd-trace-js/pull/5359/commits/27689a92ec824e74927cfefb1fe1b442e6e60289)!